### ISSUE TITLE
Issue with using axe runner

### ISSUE THREAD

**Issue by sarahughes** [pa11y/pa11y#486]
## Expected behaviour
Expecting specific axe results. 

## Actual behaviour
When running the axe runner, I'm not seeing the axe results I expect, and it appears to be running the HTML CodeSniffer instead. 

 ```
$ pa11y --runner axe -d https://www.google.com

Welcome to Pa11y
 > Running Pa11y on URL https://www.google.com
 > Debug: Launching Headless Chrome
 > Debug: Opening URL in Headless Chrome
 > Debug: Injecting HTML CodeSniffer
 > Debug: Injecting Pa11y
 > Debug: Running Pa11y on the page
 > Debug: Document title: "Google"
```

## Steps to reproduce
`$ pa11y --runner axe -d https://www.google.com`

## Environment:
```
Pa11y:      5.2.0
Node.js:    10.16.0
npm:        6.9.0
OS:         18.7.0 (darwin)
```

**Comment by josebolos** [comment]
Hi @sarahughes!

Thanks for creating this issue! Unfortunately pa11y doesn't support axe just yet - the code has been merged to `master` but we still haven't release an update that contains these changes.

I'm hoping we'll be able to release 5.3.0 before the end of the month, and I'll let you know as soon as it happens. Stay tuned!

**Comment by sarahughes** [comment]
https://github.com/pa11y/pa11y-runner-axe/ This repo doesn't work yet?

**Comment by joeyciechanowicz** [comment]
@sarahughes the repo works, and works with the latest *git* version of pa11y. We haven't yet released 5.3.0 which will include different runners as part of it.
If you clone this repo and run `pa11y` from there then it will work.

**Comment by jsa34** [comment]
Any news on possible release date?  I am just about to use pa11y, and being able to run axe tests would help immensely without having to clone the repo, but it's a workaround if needs must!

**Comment by thibaudcolas** [comment]
I’m not sure if this will work for you but I’ve been able to use Pa11y with Axe successfully by installing Pa11y from git. With npm:

```sh
npm install -g git+ssh://git@github.com:pa11y/pa11y.git#master
```

**Comment by jsa34** [comment]
Proxy issues for me using @thibaudcolas approach (I guess my org's proxy settings are blocking it for some reason... :S) But that's an issue I have to fix here!

**Comment by joeyciechanowicz** [comment]
We have just released Pa11y `5.3.0` which includes aXe!
